### PR TITLE
Avoid back and forth conversion between strings and bytes.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Signed-off-by: Anusha Ragunathan anusha@docker.com

### CODE REVIEW

**Comment by anusha-ragunathan** [comment]
@crosbymichael @tiborvass

File: cli/error.go [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+	if len(errList) < 1 {
 		return ""
 	}
-	var buf bytes.Buffer
-	buf.WriteString(errs[0].Error())
-	for _, err := range errs[1:] {
-		buf.WriteString(", ")
-		buf.WriteString(err.Error())
+
+	var out []string
```
Comment (tiborvass) [inline cli/error.go]: `out := make([]string, len(errList)`

**Comment by tonistiigi** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Comment by crosbymichael** [comment]
Should this go to a different branch @tiborvass ?

**Comment by tiborvass** [comment]
@crosbymichael nah its fine we'll cherrypick

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**